generate_errcodes_src.sh

```diff
@@ -3,7 +3,7 @@
 # Used to generate epgsql_errcodes.erl
 #
 ERRFILE="https://raw.github.com/postgres/postgres/master/src/backend/utils/errcodes.txt"
```

Comment:
Should have noticed that earlier, but should we tie it to a specific version or tag or something?

Reply:
I don't think that nakes much sense: it seems that this table is backwards-compatible. So they only add to it but not remove. It should be ok to sync with latest master from time to time

Reply:
Ok, makes sense. Thanks!